<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QWidget" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>361</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>411</width>
     <height>341</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="commTab">
       <attribute name="title">
        <string>Comms</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>381</width>
          <height>101</height>
         </rect>
        </property>
        <property name="title">
         <string>Serial Port</string>
        </property>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>16</x>
           <y>30</y>
           <width>101</width>
           <height>51</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Port</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Baud Rate</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>30</y>
           <width>231</width>
           <height>62</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QComboBox" name="portComboBox"/>
          </item>
          <item>
           <widget class="QComboBox" name="bRateComboBox"/>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="plotTab">
       <attribute name="title">
        <string>Plot</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>381</width>
          <height>121</height>
         </rect>
        </property>
        <property name="title">
         <string>Scales</string>
        </property>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>350</width>
           <height>75</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="4">
           <widget class="QDoubleSpinBox" name="maxLeftSpinBox">
            <property name="minimum">
             <double>-50.000000000000000</double>
            </property>
            <property name="maximum">
             <double>50.000000000000000</double>
            </property>
            <property name="value">
             <double>5.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="minRightSpinBox">
            <property name="maximum">
             <double>22000.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label1">
            <property name="text">
             <string>Min Left</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label2">
            <property name="text">
             <string>Min Right</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="minLeftSpinBox">
            <property name="minimum">
             <double>-50.000000000000000</double>
            </property>
            <property name="maximum">
             <double>50.000000000000000</double>
            </property>
            <property name="value">
             <double>-5.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="4">
           <widget class="QDoubleSpinBox" name="maxRightSpinBox">
            <property name="minimum">
             <double>0.000000000000000</double>
            </property>
            <property name="maximum">
             <double>22000.000000000000000</double>
            </property>
            <property name="value">
             <double>20000.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QLabel" name="label4">
            <property name="text">
             <string>Max Right</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QLabel" name="label3">
            <property name="text">
             <string>Max Left</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="2">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="calTab">
       <attribute name="title">
        <string>Cals</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>381</width>
          <height>141</height>
         </rect>
        </property>
        <property name="title">
         <string>Depth Calibration File</string>
        </property>
        <widget class="QComboBox" name="depthComboBox">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>361</width>
           <height>23</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>70</y>
           <width>221</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Send Calibration to Winch Panel</string>
         </property>
        </widget>
        <widget class="QPushButton" name="sendDepthCal">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>100</y>
           <width>99</width>
           <height>27</height>
          </rect>
         </property>
         <property name="text">
          <string>Send</string>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>170</y>
          <width>381</width>
          <height>80</height>
         </rect>
        </property>
        <property name="title">
         <string>Tension Calibration File</string>
        </property>
        <widget class="QComboBox" name="tensionComboBox">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>361</width>
           <height>23</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
